coalib/parsing/LineParser.py

```diff
@@ -62,8 +62,16 @@ def parse(self, line):
         # Escapes in value might be needed by the bears
         keys, value = self.__extract_keys_and_value(line)
 
+        all_delimiters = self.key_value_delimiters
+        all_delimiters += self.key_delimiters
+        all_delimiters += self.comment_seperators
+        all_delimiters += self.section_override_delimiters
+        all_delimiters = "".join(all_delimiters)
```

Comment:
I think then the keys and values from `section_name_surroundings` are missing too, right?